connector.py

```diff
@@ -0,0 +1,25 @@
+""" Copyright start
+  Copyright (C) 2008 - 2022 Fortinet Inc.
```

Comment:
Copyright content should be 
"""
Copyright start
MIT License
Copyright (c) 2024 Fortinet Inc
Copyright end
"""